README.md

```diff
@@ -198,6 +198,10 @@ With reference to the [GitHub Commit status API], the `POST` parameters (`state`
   The log level (one of `debug`, `info`, `warn`, `error`, `silent`).\
   Default: `info`.
 
+- `github_hostname`:\
+  Override the default GitHub hostname. This allows you to post commit statuses to repositories hosted by GitHub Enterprise (GHE) instances. e.g, `github.mycompany.org`, without protocol (only HTTPS is supported) and without API endpoint suffix (`/api/v3/`).
```

Comment:
This allows you to post commit -> This allows to post commit

Reply:
> e.g,

Beginning of new phrase, needs uppercase. If it is a line break, it should be indented below "Override".

> Override the default GitHub hostname. 

There is no need to specify the concept of overriding, otherwise we should do the same on all settings with a default. We can simply say:

> GitHub hostname. This allows ...

> without API endpoint suffix (`/api/v3/`).

Calling it suffix is super confusing, since it is the prefix of the actual endpoint. We can say

> without API path prefix (`/api/v3/`).

Even better, I would keep it simpler:

> GitHub hostname. This allows to post commit statuses to repositories hosted by GitHub Enterprise (GHE) instances. For example: `github.mycompany.org` will be expanded by cogito to `https://github.mycompany.org/api/v3`.

Reply:
Changed as suggested.